.github/workflows/publish-corfu.yml

```diff
@@ -1,15 +1,16 @@
-name: CorfDb Artifacts
-
-on:
-  push:
-    branches:
-      - master
+name: CorfuDB Artifacts
+#
+#on:
+#  push:
+#    branches:
+#      - master
+on: pull_request
```

Comment:
don't forget to change back to master, when your fix is done

Reply:
sure will do